### PR TITLE
fast tokenizer API update

### PR DESCRIPTION
**Pull request by YoelShoshan**

### CODE REVIEW

File: fusedrug/data/tokenizer/ops/fast_tokenizer_ops.py [hunk 1]
```diff
@@ -27,8 +28,10 @@ def __init__(
         Args:
             tokenizer_json: full path to a json file that the tokenizer will be loaded from
             max_size: sequences below this size will be padded, and above this size will be truncated
-            pad_id: either an int describing explicitly the token id, or a string of a token for which the token id will be looked up in the loaded tokenizer vocab
+            pad: a string of the pad token
```
Comment (mosheraboh) [inline fusedrug/data/tokenizer/ops/fast_tokenizer_ops.py:31]: pad_token

File: fusedrug/utils/misc.py [hunk 2]
```diff
@@ -15,5 +15,12 @@ def worker_init_fn(worker_id: int) -> None:
     # to ensure different seed on each worker
```
Comment (mosheraboh) [inline fusedrug/utils/misc.py:15]: I think we have all of this in fuse-med-ml already

**Review by mosheraboh — APPROVED** [review]
Looks good. minors inline.

File: fusedrug/utils/misc.py (hunk 2 above)
Comment (YoelShoshan) [reply fusedrug/utils/misc.py:15]: thanks - I can see the class Seed in FuseMedML seed.py
It's not identical, I'll compare and potentially just use the one in FuseMedML and/or modify it

**Pull request merged**